backend/views/meta_test_view.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework.views import APIView
+from rest_framework import permissions
+from views.retrieve_test_view import META_TEST, retrieve_test_data
+
+
+class MetaTestSet(APIView):
+    # Can only be accessed by Admin
+    permission_classes = [permissions.IsAdminUser]

```

Comment:
I think that this API should be open, but you should only get a full set of data back if the test you are asking about `isPublic`

Reply:
Maybe then if it's not public, but you have admin permission, you can still access it?

Reply:
I'll have to look into that more. There may be a way to see what permissions were used to access it and filter the returns accordingly. I'll have to look

Reply:
fixed in #340, which will be merged into this before going to master. Keeping them seperate for review since #340 comes with additional functionality